### ISSUE TITLE
Connect to WSL does not work

### ISSUE THREAD

**Issue by flipperdolphin** [lapce/lapce#2790]
## Lapce Version

0.3.0 and nightly

## System information

Windows 11 23H2

## Describe the bug
Trying to connect to WSL appears to do nothing. No UI appears to select a distro and no connections are made to WSL. Also, using the command pallete instead of the top left menu also results in the same no changes. There should be multiple WSL distros available on the system

## Additional information

I've tried on two different machines, both running W11 and can't seem to get a WSL window to open. I've also tried the nightly build and it doesn't seem to be fixed

If there are any logs I could pull let me know, running lapce from the command line with --wait doesn't result in any output

I've tried searching discord and GitHub issues but could not see this listed anywhere

**Comment by bradbrown-llc** [comment]
Can also confirm this behavior on W11 with v0.3.1, first time using this, freshly downloaded today. `wsl --version` output:
```WSL version: 2.0.9.0
Kernel version: 5.15.133.1-1
WSLg version: 1.0.59
MSRDC version: 1.2.4677
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22621.2861
```

**Comment by panekj** [comment]
Fixed via #2877
